src/Altinn.AccessManagement/Controllers/DelegationsController.cs

```diff
@@ -260,32 +271,32 @@ public async Task<ActionResult<List<OfferedDelegations>>> GetAllOfferedDelegatio
         /// <response code="400">Bad Request</response>
         /// <response code="500">Internal Server Error</response>
         [HttpGet]
-        [Route("accessmanagement/api/v1/delegations/getallreceiveddelegations")]
-        public async Task<ActionResult<List<ReceivedDelegation>>> GetAllReceivedDelegations([FromQuery] int coveredbyPartyId, string resourceType)
+        [Route("accessmanagement/api/v1/{who}/delegations/maskinportenschema/inbound")]
```

Comment:
Personally, I'm not a big fan of reusing the {who} parameter from Altinn 2 and the inbound/outbound naming, as compared to something like:
/{reportee}/delegations/maskinportenschema/received
/{reportee}/delegations/maskinportenschema/given
but this I guess is more of an API design discussion we are currently still having.

Reply:
Agree. The endpoint is possibly something we will change again